docs/ext/astropy_config.py

```diff
@@ -0,0 +1,27 @@
+import io
+
+from docutils import nodes
+from docutils.parsers.rst import Directive
+
+from astropy.config import generate_astropy_config
+
+
+class AstropyConfig(Directive):
+
+    def run(self):
+        buf = io.StringIO()
+        generate_astropy_config(buf)
+        text = buf.getvalue()
+        node = nodes.literal_block(text, text)
+        # self.add_name(node)
```

Comment:
Left-over from a trial?

Reply:
Yep, I wanted to see what it does but forgot about it ;)